### PR TITLE
Add meeting minutes until Meetup 1 (excluding)

### PR DESCRIPTION
**Pull request by egekorkan**
The readme explains what I have done, we will automate most of this process as explained in #8

### CODE REVIEW

File: Minutes/Readme.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- Removing mention of draft: 
+
+```html
+<h1>WoT CG at TPAC 2022</h1>
+```
+
+becomes 
+
+```html
+<h1>WoT CG at TPAC 2022</h1>
```
Comment (relu91) [inline Minutes/Readme.md:20]: The two strings are the same. Maybe you changed also the readme?

Comment (egekorkan) [reply Minutes/Readme.md:20]: Oh replace all effected it as well 😅

Comment (egekorkan) [reply Minutes/Readme.md:20]: The original state is at https://github.com/w3c/wot-cg/pull/11/commits/57533669b32810d0e5757db9d70583fb8d08c5ff  . I will change this

**Comment by egekorkan** [comment]
@relu91 other the feedback above, if you think it is fine, please merge it. Then I will contact W3C to update the minutes.

**Review by relu91 — APPROVED** [review]

**Pull request merged**